test/test_models.py

```diff
@@ -36,18 +36,20 @@ def test_mfcc(self):
 class TestMelResNet(common_utils.TorchaudioTestCase):
 
     def test_waveform(self):
+        """test the output dimensions after _MelResNet block.
```

Comment:
nit: "Validate the output dimensions of a _MelResNet block."

Reply:
Updated.
I also updated this change in #735 and #724 .